src/QueryM/Ogmios.purs

```diff
@@ -413,7 +419,7 @@ instance Show Epoch where
 
 newtype EraSummaryParameters = EraSummaryParameters
   { epochLength :: EpochLength -- 0-18446744073709552000 An epoch number or length.
-  , slotLength :: SlotLength -- <= 18446744073709552000 A slot length, in seconds.
+  , slotLength :: SlotLength -- <= 9,007,199,254,740,992 A slot length, in miliseconds.
```

Comment:
I'm not sure about leaving the mention of the range here since it's just the maximum range of `Number` and not a Cardano/Ogmios related restriction.